src/utils/mixin.js

```diff
@@ -0,0 +1,23 @@
+import CssVariables from "./cssVariables";
+
+// enums
+export const GAME_STATE_TIE = 'tie';

```

Comment:
You should not mix `""` and `''` for strings, decide for one, set it up in Prettier and auto format on file save